developer/deploying/platform.md

```diff
@@ -0,0 +1,179 @@
+# Reaction Platform
+
+Reaction Commerce offers a managed deployment option, the [Reaction Platform](http://getrxn.io/reaction-platform).
+
+The `reaction-cli` incorporates functionality for any team to deploy Reaction to multiple environments. [Visit reactioncommerce.com](http://getrxn.io/reaction-platform) to get a demo or request an **invite token**.
+
+1.  [Request invite token](http://getrxn.io/reaction-platform).
+2.  Register local environment
+3.  Create, add, and publish SSH keys
+4.  Create application environment
+5.  Deploy application
```

Comment:
For steps 2-5, we can link to the corresponding instructions in the docs below, like:

2. [Configuration](#configuration): Register local environment
3. [Keys](#keys): Create, add, and publish SSH keys
4. [Application](#applications): Create application development
5. [Deployment](#deploy): Deploy application
6. [Continuous Integration and Continuous Deployment](#cd/cdconfiguration): Optional - Set up CI/CD

Reply:
I'd stick to CI/CD in the link here (or **c47n** ;-) ), I thought about linking when I wrote,  I'm not particularly sure how well this works with Redoc and decided it'd be more efficient for me to skip.

Reply:
These kinds of links do work when clicked from the page: https://docs.reactioncommerce.com/reaction-docs/master/styleguide

What doesn't work is linking directly to a part of the page from here, like if I wanted to link you directly to React: https://docs.reactioncommerce.com/reaction-docs/master/styleguide#methods